Formula/c/cryptography.rb

```diff
@@ -23,19 +23,9 @@ class Cryptography < Formula
   depends_on "cffi"
   depends_on "openssl@3"
 
-  resource "semantic-version" do
-    url "https://files.pythonhosted.org/packages/7d/31/f2289ce78b9b473d582568c234e104d2a342fd658cc288a7553d83bb8595/semantic_version-2.10.0.tar.gz"
-    sha256 "bdabb6d336998cbb378d4b9db3a4b56a1e3235701dc05ea2690d9a997ed5041c"
-  end
-
-  resource "setuptools" do
-    url "https://files.pythonhosted.org/packages/aa/60/5db2249526c9b453c5bb8b9f6965fcab0ddb7f40ad734420b3b421f7da44/setuptools-70.0.0.tar.gz"
-    sha256 "f211a66637b8fa059bb28183da127d4e86396c991a942b028c6650d4319c3fd0"
-  end
-
-  resource "setuptools-rust" do
```

Comment:
Ok this is pretty funny: maturin itself is built using setuptools-rust (to avoid a circular dep, I assume), and since we build maturin from source, we in fact still need setuptools-rust 😂

Reply:
maybe just move to use formula instead then (that does not require any dep :) )